test_integration/mosaic.json

```diff
@@ -18,15 +18,15 @@
         "origin": {
           "anchorOrganizationAddress": "0x7de07272E4c6cb822503B46C8a41517935C3f0c0",
           "anchorAddress": "0xEa8D41fc6C6C0Ee155E5F6FbF9Cc1167Fa17927E",
-          "ostGatewayOrganizationAddress": "0xaB5335771fBA0270634e649882229c7e8CaA6158",
-          "ostEIP20GatewayAddress": "0xA7f056b1320fE619571849f138Cd1Ae2f2e64179"
+          "gatewayOrganizationAddress": "0xaB5335771fBA0270634e649882229c7e8CaA6158",
+          "eip20GatewayAddress": "0xA7f056b1320fE619571849f138Cd1Ae2f2e64179"
         },
         "auxiliary": {
           "anchorOrganizationAddress": "0xb8c982d0cd5f7c1B24B80f0e38cB25a6d9b5b716",
           "anchorAddress": "0x50c0Af1A754CE6bA25a102D4ee759b08141C4aA9",
-          "ostCoGatewayOrganizationAddress": "0xbc67Ff0116248762c940B97bA1501BeD6Fa6D093",
-          "ostPrimeAddress": "0x19F64B29789F02FFcCE2c37DFB3d65FEaDdea66a",
-          "ostEIP20CogatewayAddress": "0xB6329BcFE2050F50db2eD347b3cE67DDfAc39110",
+          "coGatewayOrganizationAddress": "0xbc67Ff0116248762c940B97bA1501BeD6Fa6D093",
+          "utilityTokenAddress": "0x19F64B29789F02FFcCE2c37DFB3d65FEaDdea66a",
+          "eip20CogatewayAddress": "0xB6329BcFE2050F50db2eD347b3cE67DDfAc39110",
```

Comment:
In mosaic.json it has been renamed to eip20CoGatewayAddress everywhere. Please update in tests as well as mosaic confgi.